osu.Game/OsuGame.cs

```diff
@@ -391,11 +386,12 @@ private void load()
             // Transfer any runtime changes back to configuration file.
             SkinManager.CurrentSkinInfo.ValueChanged += skin => configSkin.Value = skin.NewValue.ID.ToString();
 
-            LocalUserPlaying.BindValueChanged(p =>
+            playingState.BindValueChanged(p =>
             {
-                BeatmapManager.PauseImports = p.NewValue;
-                SkinManager.PauseImports = p.NewValue;
-                ScoreManager.PauseImports = p.NewValue;
+                BeatmapManager.PauseImports = p.NewValue != LocalUserPlayingStates.NotPlaying;
+                SkinManager.PauseImports = p.NewValue != LocalUserPlayingStates.NotPlaying;
+                // For scores, we need to allow imports during "Break" state else local user's scores will never be imported.
+                ScoreManager.PauseImports = p.NewValue == LocalUserPlayingStates.Playing;
```

Comment:
This is weird, can the state not transition to `NotPlaying` on gameplay completion or something?

Reply:
Can try and make that happen sure.